.ci/jobs/fleet-server-package-mbp.yml

```diff
@@ -5,7 +5,7 @@
     description: Jenkins pipeline for the Elastic Fleet Server package process
     project-type: multibranch
     concurrent: true
-#    script-path: .ci/packaging.groovy
+    script-path: .ci/packaging.groovy
```

Comment:
Should the `head-filter-regex`  be updated for this pipeline ? 

`head-filter-regex: '(main|7\.17|8\.9\.\d+|PR-.*)'`

Reply:
It should!